cub/cub/device/dispatch/kernels/transform.cuh

```diff
@@ -543,6 +536,74 @@ _CCCL_DEVICE _CCCL_FORCEINLINE static bool elect_one()
   return ::cuda::ptx::elect_sync(~0) && threadIdx.x < 32;
 }
 
+template <int BulkCopyAlignment>
+_CCCL_DEVICE void bulk_copy_maybe_unaligned(
```

Comment:
This looks like an extremely helpful function that I'd wish to be available more broadly. Do we already have plans to make this available in a more central place?

Reply:
This is what `cuda::memcpy_async` should do IMO, but I don't have the bandwidth ATM to align this implementation with `cuda::memcpy_async`. Eventually, we want something like `cub::BlockLoad` but with shared memory as destination that could encapsulate this and make it more broadly available.